lib/husky_musher/app.py

```diff
@@ -154,13 +154,17 @@ def main():
         instrument = 'daily_attestation'
         # Repeat instance number should be days since the start of the study,
         # with the first instance starting at 1.
-        repeat_instance = 1 + (datetime.today() - STUDY_START_DATE).days
+        repeat_instance = (datetime.today() - STUDY_START_DATE).days
 
-        if repeat_instance <= 0:
+        if repeat_instance < 0:
             # This should never happen!
             app.logger.warning("Failed to create a valid repeat instance")
             return ERROR_MESSAGE
 
+        if repeat_instance == 0:
+            ### TODO: Update with new language about daily attestations start.
+            return "You're enrollment is complete! Daily attestations start on September 25, 2020."
```

Comment:
"You're" => "Your"

Reply:
I'm not sure what you're flagging with the TODO here, and this is a nitpick, but we could avoid hard-coding September 25, 2020 by creating the string from the global STUDY_START_DATE variable here.

Reply:
Thank you!

Reply:
Yeah, can remove the hard-coded date!